tests/integration/test_decision_worker.py

```diff
@@ -30,6 +31,7 @@ def simulate_data_merge_worker(in_queue: queue_wrapper.QueueWrapper) -> None:
     """
     Place example merged detection into the queue.
     """
+    time.sleep(2)
```

Comment:
Why is this needed? Does the worker take time to startup?

Reply:
Not needed. Thanks for catching that, updated.